fbw-a32nx/src/systems/instruments/src/PFD/AttitudeIndicatorHorizon.tsx

```diff
@@ -382,15 +382,15 @@ class RadioAltAndDH extends DisplayComponent<{ bus: EventBus, filteredRadioAltit
                 const chosenTransalt = this.fmgcFlightPhase <= 3 ? this.transAlt : this.transAltAppr;
                 const belowTransitionAltitude = chosenTransalt !== 0 && (!this.altitude.isNoComputedData() && !this.altitude.isNoComputedData()) && this.altitude.value < chosenTransalt;
                 let size = 'FontLarge';
-                const DHValid = this.dh >= 0;
+                const DHValid = this.dh.value >= 0 && !this.dh.isNoComputedData();
```

Comment:
im not sure about this one, since it will show the RA value regardless, but only adds the DH text and becomes Green at appropiate point when DH is valid. But I suppose the behavior is consistent with not always exposing the DH

Reply:
That sounds okay. Crucially it'll mean that if you set your DA before takeoff you don't get it during takeoff, which I think is the expected behavior. Shouldn't you always be in CRZ/DES/APPR and within 200nm by the time you get your RA readings?

Reply:
Yearh thats what I couldnt figure out. The existing implementation gives you RA readings as long as you are below 2500 feet AGL after CRZ and also activates if you select RADIO and are below 2500 feet AGL regardless of flight phase

Reply:
Ahh okay behaviour seems okay. your radio altimeter will be amber until 400 feet and be green until 2500 before reaching CRZ phase, and the DH text only shows when below 2500 feet and in CRZ or later phase. I was just worried I made some new case for the Radio Altimeter 👍